static/elements/chromedash-x-meter.js

```diff
@@ -14,53 +14,46 @@ class ChromedashXMeter extends LitElement {
     super();
     this.value = 0;
     this.max = 100;
-    this.href = '#';
   }
 
   static get styles() {
     return [
       ...SHARED_STYLES,
       css`
       :host {
-        display: inline-block;
-        contain: content;
+        display: grid;
+        grid-template-columns: 65px 1fr;
+        grid-column-gap: 15px;
+        height: 1.6em;
+        font-size: 14px;
+      }
+
+      #percentage-number {
+        place-self: center;
+      }
+
+      #track {
         background: var(--barchart-background);
         border-radius: var(--border-radius);
         overflow: hidden;
-        height: 1.7em;
-        color: var(--barchart-color);
+        height: 100%;
       }
 
-      div {
+      #indicator {
         background: var(--barchart-foreground);
         height: 100%;
         white-space: nowrap;
         padding: 3px 0;
-        text-indent: 5px;
       }
-      
-      a {
-        text-decoration: none;
-        color: var(--barchart-color);
-      }
-
-      a:hover {
-        text-decoration: none;
-        color: var(--barchart-color);
-        cursor: pointer;
-      }
-
     `];
   }
 
   render() {
     return html`
-      <a href="${this.href}">
```

Comment:
Could we keep the href?  That would allow users to click on the bar or percentage number if that is the way that they remember to do it.

Reply:
Added back on the progress bar (not on the percentage number).